src/test/regress/expected/dynamic_table.out

```diff
@@ -156,24 +156,24 @@ SELECT * FROM t1 WHERE a = 2;
 (2 rows)
 
 -- test DROP DYNAMIC TABLE
-SELECT schedule, command FROM pg_task WHERE jobname LIKE 'gp_dynamic_table_refresh%' AND command LIKE '%dt0';
- schedule  |                    command                     
------------+------------------------------------------------
- 5 * * * * | REFRESH DYNAMIC TABLE dynamic_table_schema.dt0
+SELECT schedule, command FROM pg_task WHERE jobname LIKE 'gp_dynamic_table_refresh%' AND command LIKE '%dt0%';
```

Comment:
add a test case that: insert rows, wait pg_task refresh, then select * from dt0 ;

Reply:
Thank you for the suggested test case. I'm okay with that.

However, I would like to clarify that there are already cases tested the REFRESH commands in existing cases, and the pg_task executes the same command. If there were any issues with pg_task not executing the command as expected, it would indicate a bug within pg_task, not with the dynamic table functionality itself.

Reply:
Done.